xbmc/utils/SystemInfo.cpp

```diff
@@ -601,6 +603,8 @@ std::string CSysInfo::GetOsName(bool emptyIfUnknown /* = false*/)
     osName = "macOS";
 #elif defined (TARGET_ANDROID)
     osName = "Android";
+    if (CJNIContext::GetPackageManager().hasSystemFeature("android.software.leanback"))
```

Comment:
Maybe:
```suggestion
    if (CJNIContext::GetPackageManager().hasSystemFeature("android.software.leanback"))
       osName = "Android TV";
```

To avoid relying on the previously defined value